docs/fleet/machines.md

```diff
@@ -92,6 +92,11 @@ For local communication between [parts](/build/configure/parts-and-remotes/#mach
 <br>
 
 You can also access the control interface using the [Viam mobile app](/fleet/#the-viam-mobile-app), which you can find on the [App Store](https://apps.apple.com/vn/app/viam-robotics/id6451424162) and on [Google Play](https://play.google.com/store/apps/details?id=com.viam.viammobile&hl=en&gl=US).
+The remote control on the mobile app is organized by location and organization, simplifying the process of managing your fleet of machines.
+Extend this control to your team by granting them access to your organization.
+Additionally, you can search for machines in your fleet, as well as securely upload images from your phone to the Cloud, which you can view on the Viam app's [**DATA** tab](https://app.viam.com/data/) for later use.
+
+{{<gif webm_src="/manage/mobile-app-search.webm" mp4_src="/manage/mobile-app-search.mp4" alt="Using the control interface under the locations tab on the Viam mobile app" max-width="200px">}}
```

Comment:
I think putting a GIF here is a great idea. For some reason though, idk if it's a resolution thing but on my screen it almost looks like there's a mesh overlay. But I'd recommend making this larger so we can easily read the words, and that might help the strange grid-like effect I'm seeing?

RE your question about showing hand raiser vs a different robot, what do you think of using a rented Viam rover instead so it's equivalent to the GIF above that shows a viam rover in the full app?
```suggestion
{{<gif webm_src="/manage/mobile-app-search.webm" mp4_src="/manage/mobile-app-search.mp4" alt="Using the control interface under the locations tab on the Viam mobile app" max-width="320px">}}
```